forest/src/cli/chain_cmd.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::Cid;
+use jsonrpc_v2::Error as JsonRpcError;
+use log::warn;
+use rpc_client::{block, genesis, head, messages, new_client, read_obj};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub enum ChainCommands {
+    /// Retrieves and prints out the block specified by the given CID
+    #[structopt(about = "<Cid> Retrieve a block and print its details")]
+    Block {
+        #[structopt(help = "Input a valid CID")]
+        cid: String,
```

Comment:
I would just add the short option to all the arguments for subcommands on principle, wouldnt make much of a difference in this case. Its a nice to have